tripper/backends/collection.py

```diff
@@ -4,11 +4,16 @@
 # pylint: disable=protected-access,invalid-name
 from typing import TYPE_CHECKING
 
-import dlite
-
 from tripper.literal import Literal
 from tripper.utils import parse_object
 
+try:
+    import dlite
+except ImportError as exc:
+    raise ImportError(
+        "DLite is not installed.  Install it with: `pip install DLite-Python`"
```

Comment:
```suggestion
        "DLite is not installed.\nInstall it with:\n\n    pip install DLite-Python"
```
A more terminal-friendly representation of the exception message.